service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/comparator/ResourceComparatorTest.java

```diff
@@ -0,0 +1,77 @@
+package uk.nhs.adaptors.oneoneone.cda.report.comparator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.util.Date;
+
+import org.hl7.fhir.dstu3.model.ReferralRequest;
+import org.hl7.fhir.dstu3.model.ResourceType;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class ResourceComparatorTest {
```

Comment:
It's ok to use Mockito here, however it looks to me like an overkill. You can very easily create resources and set necessary dates without using mocking tools.